UploadDaemon/UploadTask.cs

```diff
@@ -1,16 +1,17 @@
-﻿using NLog;
+﻿using Newtonsoft.Json;
+using NLog;
 using System;
 using System.Collections.Generic;
 using System.IO;
 using System.IO.Abstractions;
 using System.Linq;
 using System.Threading.Tasks;
 using UploadDaemon.Archiving;
-using UploadDaemon.SymbolAnalysis;
 using UploadDaemon.Configuration;
 using UploadDaemon.Scanning;
+using UploadDaemon.SymbolAnalysis;
 using UploadDaemon.Upload;
-using System.Diagnostics;
+using static UploadDaemon.SymbolAnalysis.RevisionFileUtils;
```

Comment:
I prefer the static import, since I don't have to write  RevisionFileUtils.RevisionOrTimestamp every time then. 

Since UploadTargetFileUtils.Parse is written like this, do you think it would be okay to leave it like this? :)